lib/mcb/commands/apiv1.rb

```diff
@@ -7,3 +7,8 @@
 option :t, 'token', 'set the authorization token',
        argument: :required,
        default: 'bats'
+option :P, 'max-pages', 'maximum number of pages to request',
+       default: 200,
```

Comment:
sensible value?

Reply:
I know, seems large right? But we have 17k+ courses and we get 100 courses back per page, so ya as it turns out this is sensible.